lenskit-eval/src/main/java/org/lenskit/eval/traintest/recommend/RecommendEvalTask.java

```diff
@@ -303,13 +305,17 @@ public ConditionEvaluator createConditionEvaluator(AlgorithmInstance algorithm,
             return null;
         }
 
+        boolean useDetails = tlb != null; // writing requires details
```

Comment:
Definitely bike-shedding.
`//when not null we are writing detailed recommendation file`?